packages/components/psammead-episode-list/src/MediaIndicator.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+
+const Wrapper = styled.div`
+  display: inline-block;
+  width: ${props => props.size};
+  height: ${props => props.size};
+`;
+
+const MediaIndicator = ({ size }) => (
+  <Wrapper
+    aria-hidden="true"
+    size={size}
+    dangerouslySetInnerHTML={{
+      __html: `
+      <svg class="rounded-play-button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40">
```

Comment:
An attribute of focusable false is needed on the SVG as IE11 makes all SVG's focusable by default

Reply:
you may also need a width for when CSS is disabled so the icon isn't massive?